WalletWasabi/WebClients/ChaumianCoinJoin/WasabiClient.cs

```diff
@@ -21,6 +22,37 @@ public WasabiClient(Uri baseUri, IPEndPoint torSocks5EndPoint = null) : base(bas
 		{
 		}
 
+		#region blockchain
+
+		public async Task<IEnumerable<string>> GetFiltersAsync(uint256 bestKnownBlockHash, int count)
+		{
+			using(var response = await TorClient.SendAndRetryAsync(HttpMethod.Get,
+																	HttpStatusCode.OK, 
+			                                                       $"/api/v1/btc/blockchain/filters?bestKnownBlockHash={bestKnownBlockHash}&count={count}"))
+			{
+				if (response.StatusCode == HttpStatusCode.NoContent)
+				{
+					return Enumerable.Empty<string>();
+				}
+				if (response.StatusCode == HttpStatusCode.NotFound)
```

Comment:
If not found, an exception should be thrown. This is counter intuitive.

Reply:
Removed. Updated IndexDownloader to handle an exception being thrown for this case.